shoov/modules/custom/shoov_pusher/shoov_pusher.module

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * Code for the Pusher API integration.
+ */
+
+
+/**
+ * Trigger a Pusher event.
+ *
+ * @param $repo_id
+ * @param $event_name
+ * @param $message
```

Comment:
Yes, treat pr as wip, so feel free to change and improve
On Jun 21, 2015 4:28 PM, "Ilya" <notifications@github.com> wrote:

> In shoov/modules/custom/shoov_pusher/shoov_pusher.module
> <https://github.com/shoov/shoov/pull/63#discussion_r32893482>:
>
> > @@ -0,0 +1,49 @@
> > +<?php
> > +/**
> > + * @file
> > + * Code for the Pusher API integration.
> > + */
> > +
> > +
> > +/**
> > + * Trigger a Pusher event.
> > + *
> > + * @param $repo_id
> > + * @param $event_name
> > + * @param $message
>
> I think this line can be removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shoov/shoov/pull/63/files#r32893482>.
>